crates/runtime/wit/deps.toml

```diff
@@ -1,7 +1,14 @@
 blobstore = "https://github.com/WebAssembly/wasi-blobstore/archive/main.tar.gz"
 http = "https://github.com/WebAssembly/wasi-http/archive/v0.2.0.tar.gz"
-io = "../../../wit/deps/io"
+io = "https://github.com/WebAssembly/wasi-io/archive/v0.2.0.tar.gz"
```

Comment:
I'd expect this to not be required if all other imports depend on the same `wasi-io` version